frontend/src/metabase/search/components/SearchResult.jsx

```diff
@@ -70,37 +103,62 @@ export default function SearchResult(props) {
       return <CollectionResult collection={result} options={props} />;
     case "dashboard":
       return <DashboardResult dashboard={result} options={props} />;
+    case "table":
+      return <TableResult table={result} options={props} />;
+    case "segment":
+      return <SegmentResult segment={result} options={props} />;
+    case "metric":
+      return <MetricResult metric={result} options={props} />;
     default:
-      // metric, segment, and table deliberately included here
       return <DefaultResult result={result} options={props} />;
   }
 }
 
-const CollectionLink = styled(Link)`
-  display: flex;
-  align-items: center;
-
-  font-size: 12px;
-  line-height: 12px;
-  font-weight: bold;
-  color: ${color("text-medium")};
-
-  .Icon {
-    color: ${color("text-light")};
-  }
+function TableResult({ table, options }) {
+  return (
+    <ResultLink to={table.getUrl()} compact={options.compact}>
+      <Flex align="start">
+        <ItemIcon item={table} type="table" />
+        <Box>
+          <Title>{table.name}</Title>
+          <Text>
+            Table in &nbsp;
+            <span>
+              <Link to={Urls.browseDatabase({ id: table.database_id })}>
+                <Database.Name id={table.database_id} />{" "}
```

Comment:
```suggestion
                <Database.Name id={table.database_id} />&nbsp;
```

Some linting rules that @nemanjaglumac is updating might not like `{" "}` so can you change it to `&nbsp;`?